wear/src/main/java/io/homeassistant/companion/android/viewModels/EntityViewModel.kt

```diff
@@ -9,4 +9,9 @@ import io.homeassistant.companion.android.common.data.integration.Entity
 class EntityViewModel : ViewModel() {
 
     var entitiesResponse: Array<Entity<Any>> by mutableStateOf(arrayOf())
+    var expandedInputBooleans: Boolean by mutableStateOf(true)
+    var expandedLights: Boolean by mutableStateOf(true)
+    var expandedScenes: Boolean by mutableStateOf(true)
+    var expandedScripts: Boolean by mutableStateOf(true)
+    var expandedSwitches: Boolean by mutableStateOf(true)

```

Comment:
This is bad, these shouldn't be in the view model.  View model is only for data not the view state.